package/MDAnalysis/topology/GROParser.py

```diff
@@ -71,7 +71,7 @@ def parse(self):
                     charge = guess_atom_charge(name)
                     # segid = "SYSTEM"
                     # ignore coords and velocities, they can be read by coordinates.GRO
-                except:
+                except (ValueError, IndexError):
```

Comment:
yeah I noticed this weird issue to some time ago. I don't know where it comes from but I guess the coverage branch prediction fails here. I hope the next version will fix this.